.travis.yml

```diff
@@ -1,54 +1,22 @@
+sudo: false
 dist: trusty
 language: rust
-rust:
-  - 1.22.1
-  - nightly
+cache:
+  cargo: true
+
 matrix:
-  fast_finish: true
-  allow_failures:
-  - rust: nightly
-os:
-  - linux
-#  - osx
+  include:
+    - os: linux
+      rust: stable
 
-# Travis is slow and often backlogged so we only run it on the "auto" branch
-# which is what bors uses to test the merge commit when merging a PR. Individual
-# PRs can catch errors by checking the taskcluster result instead.
 branches:
   only:
-    - auto
-notifications:
-  webhooks: http://build.servo.org:54856/travis
+    - master
 
-addons:
-  apt:
-    sources:
-      - sourceline: 'deb http://apt.llvm.org/trusty/ llvm-toolchain-trusty-3.9 main'
-        keyurl: 'http://apt.llvm.org/llvm-snapshot.gpg.key'
-      - sourceline: 'ppa:jonathonf/python-2.7'
-    packages:
-      - libgl1-mesa-dev
-      - llvm-3.9-dev
-      - libedit-dev
-      - python
-env:
-  - BUILD_KIND=DEBUG RUST_BACKTRACE=1 RUSTFLAGS='--deny warnings'
-  - BUILD_KIND=RELEASE RUST_BACKTRACE=1 RUSTFLAGS='--deny warnings'
-before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then export LLVM_CONFIG=/usr/lib/llvm-3.9/bin/llvm-config; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install zlib; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then export PKG_CONFIG_PATH="/usr/local/opt/zlib/lib/pkgconfig:$PKG_CONFIG_PATH"; fi
-  - virtualenv ../venv
-  - source ../venv/bin/activate
-  - python --version
-  - pip install mako servo-tidy
 script:
-  - servo-tidy
-  - if [ $BUILD_KIND = DEBUG ]; then (cd webrender_api && cargo test --verbose --features "ipc"); fi
-  - if [ $BUILD_KIND = DEBUG ]; then (cd webrender && cargo build --verbose --no-default-features); fi
-  - if [ $BUILD_KIND = DEBUG ]; then (cd webrender && cargo build --verbose --features profiler,capture); fi
-  - if [ $BUILD_KIND = DEBUG ]; then (cargo test --all --verbose); fi
-  - if [ $BUILD_KIND = RELEASE ]; then (cd wrench && python script/headless.py reftest); fi
-  - if [ $BUILD_KIND = RELEASE ]; then (cd wrench && cargo build --release); fi
-  - if [ $TRAVIS_RUST_VERSION == "nightly" ]; then (cd webrender && cargo bench --verbose); fi
+  - cd webrender_api
+  - cargo build --verbose
```

Comment:
Just calling `cargo build --verbose` from the root would automatically apply `--all` that affects all the workspace projects, no need to go to each of them separately